Package.swift

```diff
@@ -65,9 +54,18 @@ let package = Package(
                 "Resources",
             ]
         ),
-    ] + (staticSwiftSyntax ? [.binaryTarget(
+        .target(
+            name: "SyntaxParserBinaryWrapper",
+            dependencies: [.target(name: "lib_InternalSwiftSyntaxParser")],
+            // Pass `-dead_strip_dylibs` to ignore the dynamic version of `lib_InternalSwiftSyntaxParser`
+            // that ships with SwiftSyntax because we want the static version from
+            // `StaticInternalSwiftSyntaxParser`.
+            linkerSettings: [.unsafeFlags(["-Xlinker", "-dead_strip_dylibs"])]
```

Comment:
Ideally we could just add this to the `.binaryTarget` but that isn't supported. https://github.com/apple/swift-package-manager/issues/5618